docs/ops/deployment/kubernetes.md

```diff
@@ -230,6 +240,8 @@ spec:
         volumeMounts:
         - name: flink-config-volume
           mountPath: /opt/flink/conf/
+        securityContext:
+          runAsUser: 9999
```

Comment:
Why securityContext need to be set? What will happen if the uid(9999) does not exist.

Reply:
I stumbled over this trying to deploy the example templates to a Kubernetes cluster where I did not have permissions to run privileged workloads. It seems to me to be a best practice to run unprivileged if possible.
The user _flink_ with uid _9999_ is present in the official Flink docker image, e.g. [here](https://github.com/docker-flink/docker-flink/blob/2e4b45b10e8efe04c324e44cacf7df16b2553f0f/1.9/scala_2.12-debian/Dockerfile#L55) for Flink 1.9/Scala 2.12.

Reply:
Maybe we should then add a comment that this refers to a user ID that exists in the Flink docker images and that users should change it if necessary.

Reply:
Agree. We should add a comment here.

Reply:
@mbode any update here?

Reply:
Whoops, must have missed it, sorry! Just pushed the comment like you proposed.

Reply:
Cool, thanks! ☕️